metrics/metrics.go

```diff
@@ -1,36 +1,66 @@
 package metrics
 
 import (
+	"fmt"
 	"log"
 	"runtime"
+	"runtime/debug"
 	"time"
 
 	"gopkg.in/Clever/kayvee-go.v6"
 	"gopkg.in/Clever/kayvee-go.v6/logger"
 )
 
-func logMetric(source, metricName, metricType string, value uint64) {
-	log.Printf(kayvee.FormatLog(source, kayvee.Info, metricName, logger.M{
-		"type":  metricType,
-		"value": value,
-		"via":   "process-metrics",
-	}))
-}
-
 // Log records Golang process metrics such as HeapAlloc, NumGC, etc... every
 // frequency time period. This function never returns so it should be called from
 // a Goroutine.
 func Log(source string, frequency time.Duration) {
-	for _ = range time.Tick(frequency) {
-		logMetric(source, "NumGoroutine", "gauge", uint64(runtime.NumGoroutine()))
+	logMetric := func(metricName, metricType string, value uint64) {
+		log.Printf(kayvee.FormatLog(source, kayvee.Info, metricName, logger.M{
+			"type":  metricType,
+			"value": value,
+			"via":   "process-metrics",
+		}))
```

Comment:
Nit.  Could use kv-logger instead of just formatter.

Reply:
Double Nit. Mind changing the log level to "Trace".  It should be available in the latest version of kv.